src/scaffoldmaker/meshtypes/meshtype_3d_wholebody2.py

```diff
@@ -0,0 +1,432 @@
+            "Number of elements across torso",
+            "Number of elements across arms",
+            "Number of elements across legs"]
+        return optionNames
+
+    @classmethod
+    def getOptionValidScaffoldTypes(cls, optionName):
+        if optionName == "Network layout":
+            return [MeshType_1d_network_layout1]
+        return []
+
+    @classmethod
+    def getOptionScaffoldTypeParameterSetNames(cls, optionName, scaffoldType):
+        if optionName == "Network layout":
+            return cls.getParameterSetNames()
+        assert scaffoldType in cls.getOptionValidScaffoldTypes(optionName), \
+            cls.__name__ + ".getOptionScaffoldTypeParameterSetNames.  " + \
+            "Invalid option \"" + optionName + "\" scaffold type " + scaffoldType.getName()
+        return scaffoldType.getParameterSetNames()
+
+    @classmethod
+    def getOptionScaffoldPackage(cls, optionName, scaffoldType, parameterSetName=None):
+        """
+        :param parameterSetName:  Name of valid parameter set for option Scaffold, or None for default.
+        :return: ScaffoldPackage.
+        """
+        if parameterSetName:
+            assert parameterSetName in cls.getOptionScaffoldTypeParameterSetNames(optionName, scaffoldType), \
+                "Invalid parameter set " + str(parameterSetName) + " for scaffold " + str(scaffoldType.getName()) + \
+                " in option " + str(optionName) + " of scaffold " + cls.getName()
+        if optionName == "Network layout":
+            if not parameterSetName:
+                parameterSetName = "Default"
+            return getDefaultNetworkLayoutScaffoldPackage(cls, parameterSetName)
+        assert False, cls.__name__ + ".getOptionScaffoldPackage:  Option " + optionName + " is not a scaffold"
+
+    @classmethod
+    def checkOptions(cls, options):
+        if not options["Network layout"].getScaffoldType() in cls.getOptionValidScaffoldTypes("Network layout"):
+            options["Network layout"] = cls.getOptionScaffoldPackage('Network layout', MeshType_1d_network_layout1)
+        for key in [
+            "Number of elements around head",
+            "Number of elements around torso",
+            "Number of elements around arms",
+            "Number of elements around legs"
+        ]:
+            if options[key] < 8:
+                options[key] = 8
+
+        for key in [
+            "Number of elements across head",
+            "Number of elements across torso",
+            "Number of elements across arms",
+            "Number of elements across legs"
+        ]:
+            if options[key] < 4:
+                options[key] = 4
+
+        if options["Number of elements through wall"] < 0:
+            options["Number of elements through wall"] = 0
+
+        if options["Use Core"]:
+            if options["Target element density along longest segment"] < 5.0:
+                options["Target element density along longest segment"] = 5.0
+        else:
```

Comment:
I don't want this higher limit when "Use Core" is True -- user can see if a combination doesn't work.

Reply:
Higher limit has been removed.